template-for-repository/proofs/run-cbmc-proofs.py

```diff
@@ -160,7 +160,7 @@ def get_proof_dirs(proof_root, proof_list, proof_marker):
 
 
 def run_build(litani, jobs):
-    cmd = [str(litani), "run-build"]
+    cmd = [str(litani), "run-build", "--exit-zero-on-fail"]
```

Comment:
Does this mean we have to update the run script (which is copied and not linked) in every repository based on litani (including repositories based on the starter kit)?  Would it not be cleaner to make this option the default for litani?

Reply:
> Does this mean we have to update the run script (which is copied and not linked) in every repository based on litani

Yes. Note that if I updated litani as you suggest below, we'd still have to post a PR bumping the submodule pointer on every repo.

> Would it not be cleaner to make this option the default for litani?

I'd rather not. I don't think returning zero on failure is the expected intuitive behavior.

Reply:
+1 on not returning zero on failures (by default).

Although, instead of adding this flag to `cmd` why can't we simply remove the `proc.returncode` check at line 169?

Reply:
After an offline discussion, we've decided that litani should actually return zero on pipeline failure. Litani users will have to bump their submodule; I'll take care of PRs for that. Closing this PR.

Reply:
(But to answer your question: proof failures are distinct from litani crashing or otherwise aborting, so we still want to catch the latter case even if proof failures return zero)

Reply:
Thanks, that makes sense.